doc-content/enterprise-only/openshift/proc-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-dashbuilder-standalone_{context}"]
+= Installing Dashbuilder Standalone using Operator
+
+Use Operator to install Dashbuilder Standalone separately from other services.
+
+.Procedure
+
+. On the *Installation* page, enter `standalone-dashbuilder` in the *Application name* field.
+. In the *Environment* field, enter `rhpam-standalone-dashbuilder`.
+. Click *Next*.
+. On the *Security* page, click *Next*.
```

Comment:
Security is configurable, is user wants to use ldap or rhsso they can.

Reply:
Ok, will add an optional step.